### PR TITLE
Add Hall of Flame Tributes

### LINKED ISSUE #36: implement Hall
`/api/getTribute`

```
[
{
"playerId": "d74a2473-1f29-40fa-a41e-66fa2281dfca",
"peanuts": 472959
}
]
```

### PR DESCRIPTION
**Pull request by Edgarware**
Interface is basically the same as Idols. Fixes #36.

### CODE REVIEW

File: blaseball_mike/models.py [hunk 1]
```diff
@@ -1215,6 +1215,24 @@ def player(self):
...
+
+    @classmethod
+    def load(cls):
+        tributes = database.get_tributes()
+        tributes_dict = OrderedDict()
+        for tribute in tributes:
+            tributes_dict[tribute['playerId']] = cls(tribute)
+        return tributes_dict
+
+    @property
```
Comment (jmaliksi) [inline blaseball_mike/models.py:1228]: you need to declare the setter

Comment (Edgarware) [reply blaseball_mike/models.py:1228]: `Idol` also doesnt have a setter for player, should I add one there as well?

**Comment by jmaliksi** [comment]
That sounds right. If you load up the REPL and try to use it, it should be
obvious if the setter doesn’t work properly as soon as you try to load
them.

File: blaseball_mike/models.py (hunk 1 above)
Comment (Edgarware) [reply blaseball_mike/models.py:1232]: OK, added for Idol and Tribute

**Review by jmaliksi — APPROVED** [review]

**Pull request merged**